sageintacctsdk/apis/ar_invoices.py

```diff
@@ -0,0 +1,29 @@
+"""
+Sage Intacct AR Invoice
+"""
+from typing import Dict
+
+from .api_base import ApiBase
+
+
+class ARInvoices(ApiBase):
+    """Class for AR Invoice APIs."""
+    def __init__(self):
+        ApiBase.__init__(self, dimension='ARINVOICE')
+
+    def get_all(self):
```

Comment:
We use `query` (https://developer.intacct.com/api/accounts-receivable/invoices/#list-invoices) instead of `readByQuery` for get_all() methods because query supports pagination. In`readByQuery` we can get 2000 items at max. If the records are >2000 then we end up missing them. Please remove off this method so that, it could use the base class get_all() method. Also, add the possible fields list in `constants.py` for `ARINVOICE` . We'll be soon cleaning up other readByQuery  in the SDK.